django_server/urls.py

```diff
@@ -24,7 +24,7 @@
 from rest_framework import routers
 
 router = routers.DefaultRouter()
-router.register(r'demo', DemoViewSet, r"demo")
+router.register(r'demo-view', DemoViewSet, r"demo-view")
```

Comment:
api/demo route was already there for custom_demo_controller . So had to change the route for DemoViewSet.